src/sentry/scripts/similarity/index.lua

```diff
@@ -27,7 +27,7 @@ This is modeled as two data structures:
 -- greater. This is wrapped in `pcall` so that we can continue to support older
 -- Redis versions while using this feature if it's available.
 if not pcall(redis.replicate_commands) then
-    redis.log(redis.LOG_DEBUG, 'Could not enable script effects replication.')
+    print('Could not enable script effects replication.')
```

Comment:
What's the motive behind this change?

Reply:
This one is not a great change but DragonflyDB doesn't have log function on redis object. Maybe there is a way to test the engine in lua to conditional call redis.log

Reply:
Maybe we should just check if redis has a log function and conditionally apply this.

Reply:
done, I check the log function to use it or not